apps/src/sites/studio/pages/courses/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import CourseOverview from '@cdo/apps/templates/courseOverview/CourseOverview';
+
+$(document).ready(showCourseOverview);
+
+function showCourseOverview() {
+  const courseData = document.querySelector('script[data-coursedata]');
+  const configCourse = JSON.parse(courseData.dataset.coursedata);
```

Comment:
This is a common pattern we've been using. There shouldn't be any reason that JSON.parse should fail (as we're creating this data using `to_json` on the dashboard side. I'm not really sure what else we could/should do if we have bad JSON here for some reason.